qiskit/circuit/quantumcircuit.py

```diff
@@ -504,6 +507,30 @@ def __str__(self) -> str:
         return str(self.draw(output="text"))
 
     def __eq__(self, other) -> bool:
+        """Check if two circuits are equal.
+        This will check .cablirations and .global_phrase
```

Comment:
for code references like `.calibrations` etc. these should be written using sphinx's referencing syntax (I think you might need the `:atttr:` one for these). You can find some examples of it in other docstrings, and you did it correctly already with `:class:.Operator` 😄

Reply:
hey @javabster 
I am looking at attr and some references are having issues.
For example, in quantumcircuit, the when adding .name with attr, 
I'm getting this error locally: qiskit-terra/qiskit/circuit/quantumcircuit.py:docstring of qiskit.circuit.quantumcircuit.QuantumCircuit.__eq__:1: WARNING: more than one target found for cross-reference 'name'
so I tried adding this code below in the python file
```
@property
    def name(self) -> str:
```
and that throws error Failed to import qiskit.compiler. where hint points to most likely due to a circular import.
in this scenario, what would be the correct way to approach this?
Just testing things, I found out I could add :attr:`name` where it will treat it as attr but won't do anything when clicking on it.

Thanks in advance!

Reply:
hi @uchoi91 you shouldn't add a new property to the file just to fix a sphinx error. I'm not sure exactly what the issue is here but I suspect it is because you are not being specific enough when specifying the name attribute in your docstring. If you just put `:attr:.name` then sphinx doesn't know which name you are trying to link to, as many classes in the terra documentation have attributes called `name`. I think you'd need to use `:attr:QuantumCircuit.name` or something. You can check out some of the other files in the repo for examples of how they use `:attr:` in their doc strings :)

Reply:
@javabster Thanks your patience as I got the changes pushed out with my latest commit. 
Not sure what the exact process is but marking resolved from my end since I did push out changes to your comment.